src/Json.Schema.ToDotNet.Cli/Options.cs

```diff
@@ -92,6 +92,13 @@ internal class Options
             Required = false)]
         public bool GenerateCloningCode { get; set; }
 
+        [Option(
+            "generate-integer-as",
```

Comment:
Very meta question here.  In terms of the option, generate-integer implies its an integer which in a way conflict with the fact it might be say a long.  Is the proper term to use here number? as in --generate-number-as which could be set to integer, long, etc.  Open to discussion this is very meta

Reply:
good question, add my inputs to the discussion:
I originally plan to use `--generate-json-integer-type-as-csharp-type = long`
This includes the full info but I think looks long so I tried to shorten it. logically I will also use `--generate-number-as` but it has a problem here because json schema has a type `number` so to avoid confusion we should avoid the word `number`,
Currently I think like `generate-json-integer-as` or some longer version that is more clear. @cfaucon let me know your take,

Reply:
Made the proposed change.